nav2_behaviors/include/nav2_behaviors/timed_behavior.hpp

```diff
@@ -238,7 +237,8 @@ class TimedBehavior : public nav2_core::Behavior
     while (rclcpp::ok()) {
       elasped_time_ = clock_->now() - start_time;
       if (action_server_->is_cancel_requested()) {
-        RCLCPP_INFO(logger_, "Canceling %s", behavior_name_.c_str());
+        result->error_msg = "Canceling " + behavior_name_;
```

Comment:
Cancel isn't an error message-able result

Reply:
removed it.